src/formatDate.js

```diff
@@ -8,7 +8,71 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  // create property for symbols from and to
+  const simbolFormat = fromFormat[fromFormat.length - 1];
+  const simbolTo = toFormat[toFormat.length - 1];
+  // create split array from date
+  const dateDay = date.split(simbolFormat);
+  // create property that will save new date
+  let year = '';
+  let day = '';
+  let month = '';
+
+  // add value for year, day, month from dateDay
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    switch (fromFormat[i]) {
+      case 'YYYY':
+        year = dateDay[i];
+        break;
+      case 'YY':
+        year = dateDay[i];
+        break;
+      case 'MM':
+        month = dateDay[i];
+        break;
+      case 'DD':
+        day = dateDay[i];
+        break;
+    }
```

Comment:
`switch/case` should always have `default` case for error handling

We don't have a checklist in this task, but I think we need to add default case to both switches

Reply:
> `switch/case` should always have `default` case for error handling
> 
> We don't have a checklist in this task, but I think we need to add default case to both switches

Good point, you can throw an error in this case